### PR TITLE
Run celery.backend_cleanup via Badpenny

### PR DESCRIPTION
**Pull request by djmitche**
@lundjordan what do you think?

### CODE REVIEW

File: relengapi/lib/celery.py [hunk 1]
```diff
@@ -58,6 +59,15 @@ def wrap(fn):
     return inner(**kwargs)
 
 
+@badpenny.periodic_task(3600 * 24)
+def backend_cleanup(js):
+    # Celery's built-in backend_cleanup task depends on Beat, which relengapi
+    # does not use, so we just run it via badpenny
+    backend_cleanup_task = current_app.celery.tasks['celery.backend_cleanup']
+    print "got herez"
```
Comment (Callek) [inline relengapi/lib/celery.py]: debugz

Comment (djmitche) [inline relengapi/lib/celery.py]: gonez, thanks :)

File: relengapi/lib/celery.py [hunk 2]
```diff
@@ -58,6 +59,14 @@ def wrap(fn):
     return inner(**kwargs)
 
 
+@badpenny.periodic_task(3600 * 24)
+def backend_cleanup(js):
+    # Celery's built-in backend_cleanup task depends on Beat, which relengapi
+    # does not use, so we just run it via badpenny
+    backend_cleanup_task = current_app.celery.tasks['celery.backend_cleanup']
+    backend_cleanup_task.apply()
```
Comment (lundjordan) [inline relengapi/lib/celery.py]: looks good.

side-note: celery src suggests this is a builtin like chord but I can't figure out how to import it outside looking it up by name in list of tasks: https://github.com/celery/celery/blob/master/celery/app/builtins.py#L33

**Comment by lundjordan** [comment]
lgtm - I noticed that beat has a check for looking if 1) we actually want tasks to expire (CELERY_TASK_RESULT_EXPIRES != None) and 2) if the backend supports autoexpire (I'm guessing for things like db backends): https://github.com/celery/celery/blob/master/celery/beat.py#L201

Do we want those checks?

I'm worried if we do not, relengapi will trip at this badpenny step for environments where devs don't set a backend, disable CELERY_TASK_RESULT_EXPIRES, or the backend is not a db backend.

**Comment by djmitche** [comment]
That's not a bad point - I could probably duplicate those checks easily.

I think getting the task from the task list is probably more durable than trying to import it.

**Comment by djmitche** [comment]
How's that?

**Comment by lundjordan** [comment]
r+ lgtm +1

**Pull request merged**